### PR TITLE
config info in doc

### PR DESCRIPTION
**Pull request by neha-Gupta1**
- Use single inverted comma instead of double inverted comma in Powershell env vars.
- Update config file information in docs.  

#### Does this PR need a docs update or release note?

- [ ] :no_entry: No

#### Type of change

- [ ] :world_map: Documentation

#### Issue(s)

* https://github.com/alcionai/corso/issues/3522

#### Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Review by ryanfkeepers — APPROVED** [review]

File: website/docs/setup/configuration.md [hunk 1]
```diff
@@ -127,6 +132,24 @@ directory within the container.
 </TabItem>
 </Tabs>
 
+Similar to environment variables, user can specify the values in config file.
```
Comment (ntolia) [inline website/docs/setup/configuration.md:135]: This section is unclear. How should this file be used? It's also outside of a Tab group and therefore is very platform specific. Please remove or add more docs and instructions for every platform.

**Review by ntolia — CHANGES_REQUESTED** [review]

File: website/docs/setup/configuration.md (hunk 1 above)
Comment (neha-Gupta1) [reply website/docs/setup/configuration.md]: Information about how to use the config file is already mentioned. And I have updated the docs in each platform respectively. Please do let me know if any other changes are required.

File: website/docs/setup/configuration.md [hunk 2]
```diff
@@ -127,6 +168,24 @@ directory within the container.
 </TabItem>
 </Tabs>
 
+The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```
Comment (ntolia) [inline website/docs/setup/configuration.md:171]: Given the additions above, shouldn't this be removed? It's outside of a tab group.

Comment (neha-Gupta1) [reply website/docs/setup/configuration.md:168]: It should have been inside docker's tab group. Moved it there.

File: website/docs/setup/configuration.md [hunk 3]
```diff
@@ -104,12 +111,46 @@ To create the environment variables file, you can run the following command:
 
 By default, Corso stores its configuration file (`.corso.toml`) in the user's home directory.
 The location of the configuration file can be specified using the `--config-file` option.
+The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```
Comment (ntolia) [inline website/docs/setup/configuration.md:114]:
```suggestion

The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```

File: website/docs/setup/configuration.md [hunk 4]
```diff
@@ -104,12 +111,46 @@ To create the environment variables file, you can run the following command:
...
+ # Corso passphrase
+ passphrase = '...'
+```
 
 </TabItem>
 <TabItem value="unix" label="Linux/macOS">
 
 By default, Corso stores its configuration file (`.corso.toml`) in the user's home directory.
 The location of the configuration file can be specified using the `--config-file` option.
+The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```
Comment (ntolia) [inline website/docs/setup/configuration.md:137]:
```suggestion

The config file can also be used to provide other configuration information like Azure and AWS credentials as mentioned below:
```

**Review by ntolia — APPROVED** [review]

**Pull request merged**